workflows/smartseq2/smartseq2_checker.wdl

```diff
@@ -0,0 +1,15 @@
+task SmartSeq2Checker {
+  File aln_metrics
+
+  String expected_aln_metrics_hash
+
+  command <<<
+    aln_metrics_hash=$(tail -n +6 ${aln_metrics} | md5sum | awk '{print $1}')
```

Comment:
You can get awk to filter out comment lines with `awk 'NF && $1!~/^#/'`, which serves this same purpose, and could be a bit more robust.